src/frontend/ReplicacheProvider.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactNode, useEffect, useState } from "react";
+import { MutatorDefs, Replicache, ReplicacheOptions } from "replicache";
+import { ReplicacheContext } from "./context.js";
+import { getPokeReceiver } from "./poke.js";
+
+interface ReplicacheProviderProps
+  extends Omit<ReplicacheOptions<MutatorDefs>, "licenseKey" | "name"> {
+  children: ReactNode;
+  name?: string;
+}
+
+export function ReplicacheProvider({
+  children,
+  name,
```

Comment:
One thing is that I think in some applications it is common to have more than one Replicache instance open at a time.

Reply:
On second thought ignore this. We should wait to add API until needed.